toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -807,6 +814,8 @@ func addEntryToFstab(installRoot, mountPoint, devicePath, fsType, mountArgs stri
 		device = devicePath
 	} else if diskutils.IsReadOnlyDevice(devicePath) {
 		device = devicePath
+	} else if "proc" == devicePath {
```

Comment:
Not sure about this special casing but I can see why it was done. Thoughts on adding a parameter to `addEntryToFstab()` to handle special mountpoints like this?

Something like `addEntryToFstab(installRoot, mountPoint, devicePath, fsType, mountArgs string, doPseudoFsMount bool)` and then change these if conditions to check for the bool. This should give us some flexibility if we need to add something similar for "/sys" in the near future.